File: 1748.py
class Solution:
    def sumOfUnique(self, nums: List[int]) -> int:
        h = dict()
        for x in nums:
            h[x] = h.get(x, 0) + 1
        res = 0
        for x, c in h.items():
            if c == 1:
                res += x
        return res

File: sol/1748/1748.cpp
struct Solution {
    int sumOfUnique(vector<int>& nums) {
        int h[101] = {0};
        for (int x : nums){
            ++h[x];
        }
        int res = 0;
        for (int i = 1; i <= 100; ++i){
            if (h[i] == 1){
                res += i;
            }
        }
        return res;
    }
};
